index.html

```diff
@@ -172,7 +186,18 @@ <h3 class="about__links">LET'S CONNECT</h3>
         <hr>
     </div>
   </section>
-  <section class="contact_form">
+  <section class="contact_form" id="contactMe">
+    <div class="eclips__dots2">
+        <div class="eclips eclips__11"></div>
+        <div class="eclips eclips__b8"></div>
+        <div class="eclips eclips__b3"></div>
+        <div class="eclips eclips__4"></div>
+        <div class="eclips eclips__12"></div>
+        <div class="eclips eclips__15"></div>
+        <div class="rectangle rectangle__7"></div>
+        <div class="rectangle__b6"></div>
+        <div class="rectangle rectange__b4"></div>
+    </div>
```

Comment:
- [ ] The `Get in touch` button for your project is different from that of the Figma design. Please fix this.

Figma
![image](https://user-images.githubusercontent.com/84156714/220799532-10863be5-450f-4190-8336-64fc26490dfb.png)

Your work
![image](https://user-images.githubusercontent.com/84156714/220799566-485eddd0-9f6f-48fc-bdf8-4e85565e40f3.png)